src/components/nav.js

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react'
+import { Box, Flex, PseudoBox, Icon } from '@chakra-ui/core'
+import Link from './link'
+
+// TODO: mobile nav fix
```

Comment:
@a-saran  why is this a entirely new file?

Reply:
Previously we have the nav component in bricks